sf-fx-runtime-java-logger/src/test/java/com/salesforce/functions/jvm/runtime/logger/DefaultLoggingFormatterTest.java

```diff
@@ -35,7 +35,7 @@ public void test_1() {
         result,
         is(
             equalTo(
-                "00:00:00.000 DEBUG [INVOCATION e3a4ae2b-fefb-4277-89d0-7068e7e39b99] foo.bar.baz - This is a message!\n")));
+                "localDateTime=00:00:00.000 level=DEBUG loggerName=foo.bar.baz message=This is a message! invocationId=e3a4ae2b-fefb-4277-89d0-7068e7e39b99 ")));
```

Comment:
I think fields that contain a space need to be quoted. Otherwise consumers have a hard time figuring out where a value with spaces starts and where it ends. Examples from https://brandur.org/logfmt support this. I couldn't find a spec to point to though.

When we implement this, I'd love to see a test-case for a message with a `"` inside it. I don't know how `logfmt` escapes, but we need to make sure we do it correctly.

Reply:
The old test tested for a newline at the end, the new one doesn't. Do we know why? Are our `logfmt` formatted lines separated with a newline?

Reply:
@Malax yeah, I can definitely add quotes for the message " ", but I'll definitely have to see if it works with the logfmt in java, I think there's not that much functionality in the java version vs go/node. 

it doesn't have a \n but I could definitely add it into the end of the stringbuilder